valhalla/baldr/graphtile.h

```diff
@@ -566,6 +582,24 @@ class GraphTile {
    */
   uint32_t turnlanes_offset(const uint32_t idx) const;
 
+  /**
+   * Convenience method to determine whether an edge is currently closed
+   * due to traffic.  Roads are considered closed when we
+   *   a) have traffic data
+   *   b) the speed is zero
+   *   c) the congestion is high
+   *
+   * If we have 0 speed, it might be that we don't have a record for
+   */
+  inline bool IsClosedDueToTraffic(const DirectedEdge* edge) const {
+    if (!traffic_tile())
+      return false;
+    auto directed_edge_index = std::distance(const_cast<const DirectedEdge*>(directededges_), edge);
+    auto live_speed = traffic_tile.getTrafficForDirectedEdge(directed_edge_index);
+    // TODO(danpat): remove the magic "4" here and define constants for these levels
```

Comment:
:+1: Should make it exportable and usable in tilecutter as well. Maybe a `is_gongestion_level_valid` function that could be called after mapping the here data.